examples/animation/unchained.py

```diff
@@ -25,7 +25,7 @@
 # Generate random data
 data = np.random.uniform(0, 1, (64, 75))
 X = np.linspace(-1, 1, data.shape[-1])
-G = 1.5 * np.exp(-4 * X * X)
```

Comment:
interesting.  That is an apparently wrong rule of thumb I have been using for a while.